streams/src/test/java/org/apache/kafka/test/KStreamTestDriver.java

```diff
@@ -44,6 +44,7 @@
 import java.util.Set;
 import java.util.regex.Pattern;
 
+@Deprecated
```

Comment:
Thanks; do you mind adding a comment recommending to use TopologyTestDriver instead, just to help out new devs coming to the project?

Reply:
Yes, I'll add it.